ios/Classes/AblyFlutterMessage.h

```diff
@@ -7,8 +7,8 @@ NS_ASSUME_NONNULL_BEGIN
 +(instancetype)new NS_UNAVAILABLE;
 -(instancetype)init NS_UNAVAILABLE;
 
--(instancetype)initWithHandle:(NSNumber *)handle
-                      message:(id)message NS_DESIGNATED_INITIALIZER;
+-(instancetype)initWithMessage:(id)message
+                      handle:(NSNumber *)handle NS_DESIGNATED_INITIALIZER;
```

Comment:
Minor but the previous signature aligned colons to keep things readable.

Reply:
Addressed in 86deac257bdc6054137ddedbfe31e85cd716412c